docs/data/data-grid/accessibility/accessibility.md

```diff
@@ -47,7 +47,7 @@ The grid responds to keyboard interactions from the user and emits events when k
 
 ### Tab sequence
 
-According to [WCAG](https://www.w3.org/TR/wai-aria-practices-1.1/#grid), only one of the focusable elements contained by the grid should be included in the page tab sequence.
+According to [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/grid/), only one of the focusable elements contained by the grid should be included in the page tab sequence.
```

Comment:
The link was broken, and since we linked WAI-ARIA which is more specific than WCAG, I have also updated the mention. See https://stereobooster.com/posts/what-are-wcag-and-wai-aria/ on the difference.